### PR TITLE
tests: linktype_name test

### PR DESCRIPTION
**Pull request by jlucovsky**
Continuation of #1853 

Issue: 6954

Ensure that the linktype_name is included in the alerts.

Updates:
- Modify existing test cases to expand the range of linktype name values.

## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6954

Suricata PR: https://github.com/OISF/suricata/pull/11564

### CODE REVIEW

**Comment by victorjulien** [comment]
Needs to pass on 6.0x and 7.0.x.

File: tests/decode-chdlc-01/test.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 requires:
 
   min-version: 6.0.0
+  lt-version: 8
```
Comment (victorjulien) [inline tests/decode-chdlc-01/test.yaml:4]: why is this needed?

File: tests/decode-chdlc-02/test.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  min-version: 8
+
+pcap: ../decode-chdlc-01/hdlc-http_1tx.pcap
+
+checks:
+
+    - filter:
+        count: 1
+        match:
+            event_type: http
```
Comment (victorjulien) [inline tests/decode-chdlc-02/test.yaml:12]: is this indentation correct yaml? Looks like 4 spaces instead of 2?

File: tests/tcp-fastopen-04/test.yaml [hunk 3]
```diff
@@ -4,6 +4,7 @@ requires:
   features:
     - HAVE_LIBJANSSON
   min-version: 5.0.0
+  lt-version: 8
```
Comment (victorjulien) [inline tests/tcp-fastopen-04/test.yaml:7]: is this needed?

Comment (victorjulien) [reply tests/tcp-fastopen-04/test.yaml:7]: same question for the other places

File: tests/decode-chdlc-02/test.yaml (hunk 2 above)
Comment (jlucovsky) [reply tests/decode-chdlc-02/test.yaml:12]: This was in the original file.

Do you recommend fixing the original in addition to this one?

File: tests/tcp-fastopen-04/test.yaml (hunk 3 above)
Comment (jlucovsky) [reply tests/tcp-fastopen-04/test.yaml:7]: will fixup

File: tests/decode-chdlc-01/test.yaml (hunk 1 above)
Comment (jlucovsky) [reply tests/decode-chdlc-01/test.yaml:4]: will remove

**Comment by jlucovsky** [comment]
Continued in #1996
